tendermint/src/lite/error.rs

```diff
@@ -1,6 +1,6 @@
 //! All error types tied to the light client.
 
-use crate::Hash;
+use crate::{account, Hash};
```

Comment:
We don't want to introduce dependencies from from the `tendermint` crate into the light module. (the other way around is OK though).